inspire/testsuite/test_bibtex.py

```diff
@@ -0,0 +1,151 @@
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+import pkg_resources
+import os
+
+from dojson.contrib.marc21.utils import create_record
+
+from invenio.testsuite import InvenioTestCase, make_test_suite, run_test_suite
+from inspire.dojson.hep import hep
+from invenio.base.wrappers import lazy_import
+Bibtex = lazy_import('inspire.utils.bibtex.Bibtex')
+
+
+class BibtexTests(InvenioTestCase):
+
+    def setUp(self):
+        self.marcxml = pkg_resources.resource_string('inspire.testsuite',
+                                                     os.path.join(
+                                                         'fixtures',
+                                                         'test_hep_formats.xml')
+                                                     )
+        record = create_record(self.marcxml)
+
+        self.hep_record = hep.do(record)
+
+        self.sample_bibtex = {
+            'collections': ('article', 'article'),
+            'citation_key': 'Aad:2015wqa',
+            'author': 'Aad, Georges',
+            'title': '\nSearch for supersymmetry in events containing a same-flavour opposite-sign dilepton pair, jets, and large missing transverse momentum in $\sqrt{s}=8$ TeV $pp$ collisions with the ATLAS detector\n',
+            'collaboration': 'ATLAS',
+            'journal': 'Eur. Phys. J.',
+            'volume': 'C75',
+            'year': 2015,
+            'number': '7',
+            'pages': '318',
+            'doi': '10.1140/epjc/s10052-015-3661-9, 10.1140/epjc/s10052-015-3518-2',
+            'note': '[Erratum: Eur. Phys. J.C75,no.10,463(2015)]',
+            'eprint': '1503.03290',
+            'archivePrefix': 'arXiv',
+            'primaryClass': 'hep-ex',
+            'reportNumber': 'CERN-PH-EP-2015-038',
+            'SLACcitation': '%%CITATION = ARXIV:1503.03290;%%',
+
+        }
+
+    def test_entry_type(self):
+        """Test if entry type is created correctly"""
+        self.assertEqual(self.sample_bibtex['collections'],
+                         Bibtex(self.hep_record)._get_entry_type())
+
+    def test_citation_key(self):
+        """Test if citation key is created correctly"""
+        self.assertEqual(self.sample_bibtex['citation_key'],
+                         Bibtex(self.hep_record)._get_citation_key())
+
+    def test_author(self):
+        """Test if author is created correctly"""
+        self.assertEqual(self.sample_bibtex['author'],
+                         Bibtex(self.hep_record)._get_author()[0])
+
+    def test_title(self):
+        """Test if title is created correctly"""
+        self.assertEqual(self.sample_bibtex['title'],
+                         Bibtex(self.hep_record)._get_title())
+
+    def test_collaboration(self):
+        """Test if collaboration is created correctly"""
+        self.assertEqual(self.sample_bibtex['collaboration'],
+                         Bibtex(self.hep_record)._get_collaboration())
```

Comment:
In these tests and the next ones it is better if you do like I mentioned before, just compute once `Bibtex(self.hep_record)` in `setUp`